step.yml

```diff
@@ -0,0 +1,172 @@
+---
+title: Create App Store Release
```

Comment:
Initially, I thought this step creates an App Store release, but it turned out It was more like creating a Release Candidate in Release Management. Might we consider updating the title to avoid confusion?